### ISSUE TITLE
0.15 Linux deb installer not creating default user correctly in all installation scenarios

### ISSUE THREAD

**Issue by EdDixon** [learningequality/installers#220]
While testing 0.15b5 develop installer on Ubuntu 14 LTS during the installation I encountered the following error:

```
sudo dpkg -i Downloads/ka-lite-bundle_0.15-b5-0ubuntu1_all.deb 
Selecting previously unselected package ka-lite-bundle.
(Reading database ... 196006 files and directories currently installed.)
Preparing to unpack .../ka-lite-bundle_0.15-b5-0ubuntu1_all.deb ...
Unpacking ka-lite-bundle (0.15~b5-0ubuntu1) ...
Setting up ka-lite-bundle (0.15~b5-0ubuntu1) ...
Setting ownership to  for assessment item database
chown: missing operand after ‘/usr/share/kalite/assessment/khan’
Try 'chown --help' for more information.
dpkg: error processing package ka-lite-bundle (--install):
 subprocess installed post-installation script returned error exit status 1
Errors were encountered while processing:
 ka-lite-bundle
```

**Comment by EdDixon** [comment]
sudo users reveals only my account so I am retitling the issue account not created...

**Comment by EdDixon** [comment]
@benjaoming is working on a quick fix for this in issue #221
